lib/sigstore/trusted_root.rb

```diff
@@ -76,6 +88,10 @@ def tlog_keys(tlogs)
 
       tlogs.each do |transparency_log_instance|
         key_bytes = transparency_log_instance.public_key.raw_bytes
+
+        # TODO: discover if this is only an issue with the key in the staging trusted root
```

Comment:
@woodruffw help figuring this out would be appreciated, as our resident cryptography expert

Reply:
Just recording: we triaged this IRL and the bug here was that `OpenSSL::PKey.read` was attempting to interpret the DER as `PKey::DH` instead of `PKey::RSA`.